src/takeLastWhile.ts

```diff
@@ -0,0 +1,46 @@
+import { purry } from './purry';
+
+/**
+ * Returns elements from the end of the array until the predicate returns false.
+ *
+ * @param array the array
+ * @param fn the predicate
+ * @signature
+ *    R.takeLastWhile(array, fn)
+ * @example
+ *    R.takeLastWhile([1, 2, 3, 4, 3, 2, 1], x => x !== 4) // => [3, 2, 1]
+ * @dataFirst
+ * @category Array
+ */
+export function takeLastWhile<T>(
+  array: ReadonlyArray<T>,
+  fn: (item: T) => boolean
+): Array<T>;
+
+/**
+ * Returns elements from the end of the array until the predicate returns false.
```

Comment:
The description needs to be clearer about what order the returned items are in. It can be read as though they are returned in reverse order (as they are checked by the predicate).

Reply:
Good point. :+1: Done. I also changed the example to a bit more obvious one (the old example repeated the same numbers in different order).